akka-http2-support/src/main/scala/akka/http/impl/engine/http2/framing/Http2FrameParsing.scala

```diff
@@ -25,7 +25,7 @@ private[http] object Http2FrameParsing {
       if (payload.hasRemaining) {
         val id = payload.readShortBE()
         val value = payload.readIntBE()
-        if (isKnownId(id)) readSettings(Setting(SettingIdentifier.byId(id), value) :: read)
+        if (SettingIdentifier.isKnownId(id)) readSettings(Setting(SettingIdentifier.byId(id), value) :: read)
```

Comment:
I wonder if the `import Http2Protocol.FrameType._` shouldn't be scoped better to avoid this in the future

Reply:
I agree that would probably make things clearer

Reply:
fdf9a75

Reply:
Good point :+1: